wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -43,6 +43,11 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>Streaming capabilities.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="AudioClipCapabilities" type="tr2:AudioClipCapabilities">
```

Comment:
Can we add mandatory element into already released command? Don't we need minOccurs="0" here?

Reply:
yes it is needed. PR is updated now with the suggested changes.